charts/jobs/templates/_helpers.tpl

```diff
@@ -69,7 +69,8 @@ Create the name of the service account to use
 {{- end -}}
 
 {{- define "flatten-name" -}}
-{{ printf "job-%s" (. | replace "." "-" | replace "/" "-") }}
+{{- $res := regexReplaceAll "[()/_-]{1}" . "" -}}
+{{- regexReplaceAll "[|.]{1}" $res "-" | trimAll "-" | lower -}}
```

Comment:
Should these end with `-}}` or just `}}`?

Reply:
The `-}}` will remove `\n`